.github/workflows/ci-rust.yml

```diff
@@ -91,15 +91,15 @@ jobs:
           RUST_LOG: debug
         timeout-minutes: 10
 
-      - name: Check non bindings crates using sodium crypto features
+      - name: Check platform crates using sodium crypto features
         run: cargo check ${{ env.CARGO_CI_FLAGS }} ${{ steps.crates.outcome.agnostic }} ${{ steps.crates.outcome.platform }} --features use-sodiumoxide
         timeout-minutes: 10
         env:
           RUST_LOG: debug
 
       # Use sodiumoxide here given 1) it is composed of C code, so not totally
       # platform independant and 2) it is what is going to be used in release
-      - name: Test Rust codebase (🐧 Linux specific)
+      - name: Test Platform Rust codebase using sodium crypto features (🐧 Linux specific)
```

Comment:
"Test platform crates using ..." to have consistant wording?

Reply:
Done